cli/lib/localization.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+const toml = require('@iarna/toml');
+const osLocale = require('os-locale');
+
+module.exports = translate;
+
+let translations = null;
+let locale = null;
+
+const replace = (str, map) =>
+  str.replace(/\{{2}(\S+?)\}{2}/gi, (_, key) => {
```

Comment:
Also could you add a test for this function? This area has a lot of churn so I'm worried someone might come along and change it later and introduce a bug, if there aren't tests.